src/platform/forms/components/review/FormSaveErrorMessage.jsx

```diff
@@ -78,8 +79,9 @@ function FormSaveErrorMessage(props) {
           We’re working to fix the problem. Please make sure you’re connected to
           the Internet, and then try saving your {appType} again.
         </p>
-        {saveLink}.
-        {!user.login.currentlyLoggedIn && (
+        {user.login.currentlyLoggedIn ? (
+          <>{saveLink}.</>
+        ) : (
```

Comment:
We were always rendering this period, despite the fact that `saveLink` does not render itself when the user is not logged in. 

Since the very next line here checks if the user ISN'T logged in and then shows a warning, I combined the two elements into a single `if user logged in ... else`. This removes the '.' from view entirely unless we're showing the logged in message and everything behaves as intended.